engine/schema/src/org/apache/cloudstack/storage/datastore/db/PrimaryDataStoreDaoImpl.java

```diff
@@ -53,6 +53,7 @@
     protected final SearchBuilder<StoragePoolVO> DcPodSearch;
     protected final SearchBuilder<StoragePoolVO> DcPodAnyClusterSearch;
     protected final SearchBuilder<StoragePoolVO> DeleteLvmSearch;
+    protected final SearchBuilder<StoragePoolVO> DcLocalStorageSearch;
```

Comment:
By convention, variables do not typically start with a capital letter.

I realize here that you are likely just doing that to fit in with the other nearby code.

This might be a nice opportunity to rename all six of these member variables to start with a lower-case version of the applicable letter (possibly even prefixing them with an underscore as many people recognize that as a convention for a member variable).